### PR TITLE
docs(event-preview): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-562.gearboxjs.preview.cogniteapp.com

File: src/components/EventPreview/EventPreview.tsx [hunk 1]
```diff
@@ -22,19 +22,37 @@ const LoadingSpinner: React.SFC = () => (
...
+  /**
+   * Function triggered when user clicks on the 'Explore event details' button. If the function is not provided the button will not be rendered.
+   */
   onShowDetails?: (e: CogniteEvent) => void;
+  /**
+   * Object map with strings to customize/localize text in the component
+   */
   strings?: PureObject;
+  /**
+   * List of event properties to be hidden. Possible values: type, subtype, description, startTime, endTime, metadata
```
Comment (mrtraser) [inline src/components/EventPreview/EventPreview.tsx]: possible values defined by type here, we can make description a bit shorter

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-562.gearboxjs.preview.cogniteapp.com

File: src/components/EventPreview/EventPreview.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ export interface EventPreviewProps {
    */
   strings?: PureObject;
   /**
-   * List of event properties to be hidden. Possible values: type, subtype, description, startTime, endTime, metadata
+   * List of event properties to be hidden. Possible values:
```
Comment (mrtraser) [inline src/components/EventPreview/EventPreview.tsx]: List of event properties to be hidden. Possible values: ->
List of event properties to be hidden

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-562.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-562.gearboxjs.preview.cogniteapp.com

**Pull request merged**